src/test/java/duke/tasks/TaskListTest.java

```diff
@@ -0,0 +1,16 @@
+package duke.tasks;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TaskListTest {
+    @Test
+    public void test2() {
```

Comment:
perhaps rename it to a more descriptive name ...something like `testcase_scenario_outcome`